tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import csv
+
+# import numpy as np
+import os
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, Tuple, Union
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Catalog, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+DATASET_NAME = "VGGFace2"
+_SEGMENTS = ("train", "test")
+
+
+CLASS_ATTRS = ("Sample_num", "Flag", "Gender")
+
+BOX2D_ATTRS = (
+    "01-Male",
+    "02-Black_Hair",
+    "03-Brown_Hair",
+    "04-Gray_Hair",
+    "05-Blond_Hair",
+    "06-Long_Hair",
+    "07-Mustache_or_Bread",
+    "08-Wearing_Hat",
+    "09-Eyeglasses",
+    "10-Sunglasses",
+    "11-Mouth_Open",
+)
+
+KEYPOINTS = ("L_eye", "R_eye", "Nose", "L_mouth_corner", "R_month_corner")
```

Comment:
why we need to set `KEYPOINTS` as a constant?

Reply:
I think it's a more convenient way to show what each coordinate of keypoints means, while it just be used once zzz.